### PR TITLE
Pages: add starter for Qwik

### PR DESCRIPTION
**Pull request by danielbayley**
See [Qwik/static-site-generation](https://qwik.builder.io/docs/guides/static-site-generation).

Based heavily on the [`astro.yml`](https://github.com/actions/starter-workflows/blob/main/pages/astro.yml) workflow.

### CODE REVIEW

**Comment by danielbayley** [comment]
> This pull request has become stale and will be closed automatically within a period of time. Sorry about that.

Bump

**Comment by yoannchaudet** [comment]
Just like in https://github.com/actions/starter-workflows/pull/2255#issuecomment-2198378233, unfortunately, we are not accepting new starter workflows for Pages at the moment.

File: pages/qwik.yml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
+    needs: build
+    runs-on: ubuntu-latest
+    name: Deploy
+    steps:
+      - name: Deploy to GitHub Pages
+        id: deployment
+        uses: actions/deploy-pages@v3
```
Comment (jsoref) [inline pages/qwik.yml:85]: There's a v4 ... https://github.com/actions/deploy-pages/releases/tag/v4.0.0

**Comment by jsoref** [comment]
@thyeggman ?

**Pull request closed without merging**